src/components/date-picker/DatePicker.scss

```diff
@@ -109,3 +109,7 @@
         }
     }
 }
+
+.tooltip {
+    @extend .bdl-Tooltip;
```

Comment:
There were a couple options to maintain backwards compatibility:
1. append the namespaced classnames e.g. `.tooltip, .bdl-Tooltip`
2. `@extend` the old classnames

I tested out both & picked the `@extend` approach, because there are a lot more special cases that the script would have to account for in option 1. (I.e. `.btn.tooltip` should be `.btn.tooltip, .btn.bdl-Tooltip` as opposed to `.btn.tooltip, .bdl-Tooltip` - there's a lot of variance like that).